video_loader.py

```diff
@@ -191,3 +234,32 @@ def _save_uri_to_tmp_file(self, uri, tmpdir):
             with open(tmp_fn, 'wb') as f:
                 f.write(binary_fn.read())
         return tmp_fn
+
+    def _process_subtitles(self, subtitle_file):
```

Comment:
Since we are not keeping the original texts from the subtitle files, this part needs to be documented in the README.
```suggestion
    def _process_subtitles(self, subtitle_file):
```